main.js

```diff
@@ -0,0 +1,47 @@
+let books = [];
+
+const booksContainer = document.getElementById('books-cont');
+function removeBook(index) {
+  window.localStorage.setItem('books', JSON.stringify(books));
+  books.splice(index, 1);
```

Comment:
- [ ] The removeBook function works fine, but if I remove a book and immediately reload the page, the book returns to the page because the setItem method that 'refreshes' the localStorage works before the book deletion. I suggest swapping the order of these lines so this behavior does not occur.